pkg/appgw/ingress_rules_test.go

```diff
@@ -77,6 +86,41 @@ var _ = Describe("MutateAppGateway ingress rules, listeners, and ports", func()
 		})
 	})
 
+	Context("ingress rules with annotated certificates", func() {
```

Comment:
Let's also add a functional test or a unit test for config builder to check the ssl cert is correctly set in the appgw config.

Reply:
> Let's also add a functional test or a unit test for config builder to check the ssl cert is correctly set in the appgw config.

Added a test case to make sure http settings is configured with ssl certificate rightly when use annotation appgw-ssl-certificate